server/index.js

```diff
@@ -1,7 +1,10 @@
+// change line to select db
+const database = 'mongo';
+

```

Comment:
I really like this and think this was part of the assignment that I didn't really take advantage of. Great job!